src/content/docs/hi/develop-and-build.mdx

```diff
@@ -0,0 +1,145 @@
+      npm run build
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm build
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run build
+      ```
+      </TabItem>
+    </Tabs>
+
+एस्ट्रो एक अलग फ़ोल्डर (डिफ़ॉल्ट रूप से `dist/`) में आपकी साइट का एक डिप्लॉय-रेडी वर्शन बनाएगा और आप टर्मिनल में इसकी प्रगति देख सकते हैं। यह आपको उत्पादन में डिप्लॉय करने से पहले आपके प्रोजेक्ट में किसी भी बिल्ड एर्रोर्स के बारे में सचेत करेगा। यदि टाइपस्क्रिप्ट को `strict` या `strictest` पर कॉन्फ़िगर किया गया है, तो `build` स्क्रिप्ट आपके प्रोजेक्ट को टाइप त्रुटियों के लिए भी जाँचेगी।
+
+जब बिल्ड समाप्त हो जाए, तो अपने टर्मिनल में उपयुक्त `प्रीव्यू` कमांड (जैसे `npm run preview`) चलाएं और आप अपनी साइट का निर्मित वर्शन उसी ब्राउज़र प्रीव्यू विंडो में स्थानीय रूप से देख सकते हैं।
+
+ध्यान दें कि यह आपके कोड का प्रीव्यू अर्थात पूर्वावलोकन  करता है जैसा कि बिल्ड कमांड को अंतिम बार चलाने पर मौजूद था। इसका उद्देश्य आपको यह प्रीव्यू देना है कि वेब पर तैनात होने पर आपकी साइट कैसी दिखाई देगी। निर्माण के बाद आपके द्वारा अपने कोड में किए गए कोई भी बाद के परिवर्तन तब तक आपकी साइट का प्रीव्यू करते समय दिखाई नहीं देंगे जब तक कि आप बिल्ड कमांड को फिर से नहीं चलाते।
+
+प्रीव्यू से बाहर निकलने के लिए (<kbd>Ctrl</kbd> + <kbd>C</kbd>) का उपयोग करें और एक अन्य टर्मिनल कमांड चलाएं, जैसे कि डेव सर्वर को पुनः आरंभ करके [डेवलपमेंट मोड में कार्य करना](#डेवलपमेंट-मोड-में-कार्य-करें) पर वापस जाएं, जो आपके संपादन के साथ अपडेट होता है और आपके कोड परिवर्तनों का लाइव पूर्वावलोकन दिखाता है।
+
+<ReadMore>[एस्ट्रो सीएलआई](/hi/reference/cli-reference/) और टर्मिनल कमांड के बारे में अधिक पढ़ें, जिनका उपयोग आप एस्ट्रो के साथ निर्माण करते समय करेंगे।</ReadMore>
+
+:::टिप
+आप बहुत ज़्यादा कोड जोड़ने या बदलने से पहले [अपनी नई साइट को तुरंत तैनात करना](/hi/guides/deploy/) चाह सकते हैं। यह आपकी साइट का न्यूनतम, कार्यशील संस्करण प्रकाशित करने में सहायक है और बाद में अपनी तैनाती के समस्या निवारण में आपका अतिरिक्त समय और प्रयास बचा सकता है।
+:::
+
+## अगले कदम
+
+सफलता! अब आप एस्ट्रो के साथ निर्माण शुरू करने के लिए तैयार हैं! 🥳
+
+यहाँ कुछ चीजें हैं जिन्हें हम आगे पढ़ने की सलाह देते हैं। आप उन्हें किसी भी क्रम में पढ़ सकते हैं। आप हमारे दस्तावेज़ों को थोड़ी देर के लिए छोड़ सकते हैं और अपने नए एस्ट्रो प्रोजेक्ट कोडबेस में खेल सकते हैं, जब भी आपको कोई परेशानी हो या कोई सवाल हो तो यहाँ वापस आएँ।
+
+### अपना डेव एनवायरनमेंट कॉन्फ़िगर करें
+
+अपने डेवलपमेंट अनुभव को अनुकूलित करने के लिए नीचे दिए गए मार्गदर्शिकाओं का अन्वेषण करें।
+
+<CardGrid>
+  <LinkCard
+    title="Editor Setup"
+    description="Customize your code editor to improve the Astro developer experience and unlock new features."
+    href="/hi/editor-setup/"
+  />
+  <LinkCard
+    title="Dev Toolbar"
+    description="Explore the helpful features of the dev toolbar."
+    href="/hi/guides/dev-toolbar/"
+  />
+  <LinkCard
+    title="TypeScript Configuration"
+    description="Configure options for type-checking, IntelliSense, and more."
+    href="/hi/guides/typescript/"
+  />
```

Comment:
These properties on the LinkCard components can and should be translated (except `href`)